script/makefiles/cluster-kind.mk

```diff
@@ -53,6 +53,15 @@ ${ADDITIONAL_CLUSTER_CONFIG}: devel/dex.crt
 
 additional-cluster-kind: ${ADDITIONAL_CLUSTER_CONFIG}
 
+devel/additional-nginx:
+	kubectl apply --kubeconfig=${ADDITIONAL_CLUSTER_CONFIG} -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+	# TODO: need to add wait for condition=exists or similar - https://github.com/kubernetes/kubernetes/issues/83242
```

Comment:
Yes, we do that in our `cluster-kind` target which installs ingress-nginx:

https://github.com/vmware-tanzu/kubeapps/blob/e5712162532c5557d98a97780eb5f73d8882fd78/script/makefiles/cluster-kind.mk#L28-L31